installers/mac/build-ballerina-macos-x64.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+function printUsage() {
+    echo "Usage:"
+    echo "$0 [options]"
+    echo "options:"
+    echo "    -v (--version)"
+    echo "        version of the ballerina distribution"
+    echo "    -p (--path)"
+    echo "        path of the ballerina distributions"
+    echo "    -d (--dist)"
+    echo "        ballerina distribution type either of the followings"
+    echo "        If not specified both distributions will be built"
+    echo "        1. ballerina"
+    echo "        2. ballerina-runtime"
+    echo "eg: $0 -v 1.0.0 -p /home/username/Packs"
+    echo "eg: $0 -v 1.0.0 -p /home/username/Packs -d ballerina"
+}
+
+BUILD_ALL_DISTRIBUTIONS=false
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case ${key} in
+    -v|--version)
+    BALLERINA_VERSION="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    -p|--path)
+    DIST_PATH="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    -d|--dist)
+    DISTRIBUTION="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    *)    # unknown option
+    POSITIONAL+=("$1") # save it in an array for later
+    shift # past argument
+    ;;
+esac
+done
+
+if [ -z "$BALLERINA_VERSION" ]; then
+    echo "Please enter the version of the ballerina pack"
+    printUsage
+    exit 1
+fi
+
+if [ -z "$DIST_PATH" ]; then
+    echo "Please enter the path of the ballerina packs"
+    printUsage
+    exit 1
+fi
+
+if [ -z "$DISTRIBUTION" ]; then
+    BUILD_ALL_DISTRIBUTIONS=true
+fi
+
+BALLERINA_DISTRIBUTION_LOCATION=${DIST_PATH}
+BALLERINA_PLATFORM=ballerina-macos-${BALLERINA_VERSION}
+BALLERINA_INSTALL_DIRECTORY=ballerina-${BALLERINA_VERSION}
+
+echo "Build started at" $(date +"%Y-%m-%d %H:%M:%S")
+
+function deleteTargetDirectory() {
+    echo "Deleting target directory"
+    rm -rf target
+}
+
+function extractPack() {
+    echo "Extracting the ballerina distribution, " $1
+    rm -rf target/original
+    mkdir -p target/original
+    unzip $1 -d target/original > /dev/null 2>&1
+
+    for jarFile in $(find target/original/ballerina-macos-$BALLERINA_VERSION -name '*.jar'); do
+        mkdir temp
+        cd temp
+        file_name=$(echo "$jarFile" | sed "s/.*\///")
+        cp ../$jarFile .
+        unzip -o $file_name
+
+        code=1
+        for lib in $(find ../temp -name '*.jnilib' -o -name '*.dylib'); do
```

Comment:
@keizer619 Should this for loop be removed?

Reply:
Yes lets remove this as well